src/components/modal-component/shared-modal.jsx

```diff
@@ -3,29 +3,33 @@ import Modal from "react-bootstrap/Modal";
 import React from "react";
 import Button from "react-bootstrap/Button";
 import { Link } from "react-router-dom";
+import './shared-modal.css'
 
 export const SharedModal = (props) => {
   const property = props.property;
   const handleCloseModal = () => props.setShow(false);
+  const checkLocation = window.location.pathname === "/";
 
   return (
     <Modal className="shared-modal" show={props.show} onHide={handleCloseModal}>
-      <Modal.Header className=" border-0" closeButton>
+        <Modal.Header
+            className={checkLocation ? "modal-header modal-primary border-0" : "modal-header modal-secondary border-0"}
+            closeButton closeVariant='white'>
         <Modal.Title>{property}</Modal.Title>
       </Modal.Header>
-      <Modal.Body>You want to {property} as</Modal.Body>
-      <Modal.Footer className="modal-footer border-0">
+      <Modal.Body className="modal-body">You want to {property} as</Modal.Body>
+      <Modal.Footer className= {checkLocation ? "modal-footer-primary": "modal-footer-secondary"}>
```

Comment:
Is any specific reason for using the checklocation condition for rendering the different modal with heading and footers?

Reply:
I added these to have different colors for different page, but have conversation with Palvi and don't have to do that. Working on that and this will be reverted